tsdb/points_test.go

```diff
@@ -785,6 +785,17 @@ func TestParsePointQuotedTags(t *testing.T) {
 	)
 }
 
+func TestParsePointsUnbalancedQuotedTags(t *testing.T) {
+	pts, err := tsdb.ParsePointsString("baz,mytag=\"a x=1 1441103862125\nbaz,mytag=a z=1 1441103862126")
+	if err != nil {
+		t.Fatalf("ParsePoints failed: %v", err)
+	}
+
+	if exp := 2; len(pts) != exp {
```

Comment:
Because `"` are often used to encapsulate stuff, can you expand this test case to show that the tag value for point 1 includes the quote, just so it's crystal to next guy e.g. me?